ui/src/app.tsx

```diff
@@ -797,7 +807,7 @@ function RoutedApp() {
           <Scheduler />
         </TooltipProvider>
         <LureAutojoiner />
-        {showDevTools && (
+        {false && (
```

Comment:
you can always just call `toggleDevTools()` from the console, maybe we should make the toggle persist so this is less annoying